webpack/assets/javascripts/react_app/components/common/CopyToClipboard/CopyToClipboard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { OverlayTrigger, Tooltip } from 'patternfly-react';
+import { translate as __ } from '../../../common/I18n';
+
+const CopyToClipboard = ({ valueToCopy }) => {
+  const inputRef = useRef(null);
+  const tooltip = <Tooltip id="copy-btn-tooltip">{__('Copied!')}</Tooltip>;
+
+  const handleCopy = () => {
+    inputRef.current.select();
+    document.execCommand('copy');
+  };
+
+  return (
+    <div className="input-group">
+      <input
+        className="form-control"
+        id="input_to_copy"
+        readOnly
+        type="text"
+        value={valueToCopy}
+        ref={inputRef}
+      />
+      <div className="input-group-btn">
+        <OverlayTrigger placement="top" overlay={tooltip} trigger="click">
```

Comment:
Can you add `rootClose`:
`<OverlayTrigger placement="top" overlay={tooltip} trigger="click" rootClose> `
So the tooltip will close when there is a click outside of it?